docs/source/python_tutorials/images/fsa_invert.svg

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
```

Comment:
Just a question: why we don't make picture above `fsa_aux.png` and `fsa_aux_symbols.png` as svg format?

Reply:
It is feasible.